### PR TITLE
IoT testsuite refactor

### PR DESCRIPTION
**Pull request by obabec**

### CODE REVIEW

**Comment by obabec** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by obabec** [comment]
@enmasse-ci run tests profile=shared-iot test_only

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot test_only

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot test_only

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot testcase=io.enmasse.systemtest.iot.shared.mqtt.MqttAdapterTest,io.enmasse.systemtest.iot.isolated.registry.FileDeviceRegistryTest

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot testcase=io.enmasse.systemtest.iot.shared.mqtt.MqttAdapterTest,io.enmasse.systemtest.iot.isolated.registry.FileDeviceRegistryTest test_only

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot testcase=io.enmasse.systemtest.iot.shared.mqtt.MqttAdapterTest,io.enmasse.systemtest.iot.isolated.registry.FileDeviceRegistryTest test_only

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot testcase=io.enmasse.systemtest.iot.shared.mqtt.MqttAdapterTest,io.enmasse.systemtest.iot.isolated.registry.FileDeviceRegistryTest test_only

**Comment by obabec** [comment]
@enmasse-ci run tests profile=shared-iot test_only

**Comment by obabec** [comment]
@enmasse-ci run tests profile=isolated-iot test_only

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot

File: systemtests/src/main/java/io/enmasse/systemtest/manager/IsolatedIoTManager.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    public void setup() throws Exception {
+        if (!kubernetes.namespaceExists(iotProjectNamespace)) {
+            kubernetes.createNamespace(iotProjectNamespace);
+        }
+    }
+
+    @Override
+    public void tearDown(ExtensionContext context) throws Exception {
+        try {
+            if (Environment.USE_MINUKUBE_ENV.equals("false")) {
```
Comment (famartinrh) [inline systemtests/src/main/java/io/enmasse/systemtest/manager/IsolatedIoTManager.java]: this will allways evaluate to false

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot

**Comment by obabec** [comment]
@enmasse-ci run tests profile=iot

**Comment by obabec** [comment]
Created new PR related to this #3262

**Pull request closed without merging**